### PR TITLE
docs: documenting new decisions

### PR DESCRIPTION
**Pull request by jinder1s**
These are some of the decisions made during/after poc of OEP-37: https://openedx.atlassian.net/wiki/spaces/AC/pages/2513076453/OEP-37+Test+Data+implementation

### CODE REVIEW

File: oeps/oep-0037/decisions/0001-yaml-centralization.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Specifically, this ADR pertains to question of where to put yaml files specifiying what data to load into the devstack development environment. As it is implemented, each IDA will have its own yaml files which lay out what data to create for that IDA.
+
+
+Decision
+--------
+
+Since this method is meant to be used in devstack to orchestrate dev data across multiple IDAs, we decided to put all of the data yaml files into the devstack repo. This decision is especially relavant for yaml files specifiying what data to create during provisioning and for very specific common workflows. The hope is that developers will only have to modify yaml files in devstack repo to create more custom data.
```
Comment (timmc-edx) [inline oeps/oep-0037/decisions/0001-yaml-centralization.rst:20]: [suggestion] Probably want to include something about the following issues with localizing the data to the IDA repos:

- Having mismatched devstack and IDA repos checked out could result in problems parsing and loading the data (including lots of irritating debugging sessions) when devstack's loader doesn't match an IDA's data or two IDAs have incompatible data
- ...and I think there was another reason, but I can't remember it. :sweat_smile:

Comment (jinder1s) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:20]: Clarification: devstack's loader really only passes the yaml files to `load_dev_data` management commands in each IDA. So this bullet point doesn't completely apply.

For reference,[ line which passes yaml file to management command](https://github.com/edx/devstack/blob/9fed63d386fedbe5da897e82cd66325ea1e12598/load_dev_data.py#L23)

Comment (timmc-edx) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:20]: Ah, OK. So it's really just that two IDAs might be out of sync with each other. (Or devstack might refer to a file that doesn't exist yet in the checked out version of the IDA, I suppose.)

Comment (jinder1s) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:20]: The worry of two repos being out of sync with each other is true for this project and in devstack usage in general. Here, the extra sync problem would arise if the yaml file in devstack has a model name that is not yet supported in user's checked out version of a IDA.

Comment (timmc-edx) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:20]: Makes sense.

File: oeps/oep-0037/decisions/0003-data-creation-functions.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-------
+
+Devstack currently has no standard way of creating dev data. Over time, developers have figured out their own ways to create data for local development. Recently, `OEP-37`_ was proposed, which suggests a standardized method for creating dev data.
+
+Decision
+--------
+
+To be iterative in implementation, we decided not to follow `OEP-37`_ completely. Specifically, we decided to implement dev data generation functions in the `load_dev_data` management command. This results in centralization of dev data creation into the `load_dev_data` management command. This is meant to be a temporary measure until we have wider usage of this method of creating data.
+
+In the future, we hope to move these dev data generation functions into aplugin style archetecture, where `load_dev_data` management command is able to discover dev data generation functions during run time.
```
Comment (timmc-edx) [inline oeps/oep-0037/decisions/0003-data-creation-functions.rst:19]:
```suggestion
In the future, we hope to move these dev data generation functions into a plugin style architecture, where `load_dev_data` management command is able to discover dev data generation functions during run time.
```
nit: typos

File: oeps/oep-0037/decisions/0003-data-creation-functions.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Consequences
+------------
+
+- Dev data generation is centralized to `load_dev_data`management command in each IDA
```
Comment (timmc-edx) [inline oeps/oep-0037/decisions/0003-data-creation-functions.rst:24]:
```suggestion
- Dev data generation is centralized to `load_dev_data` management command in each IDA
```
nit: typo

File: oeps/oep-0037/decisions/0001-yaml-centralization.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
+Centralization of Development Data Yaml files
+=============================================
+
+Status
+------
+
+Draft
```
Comment (timmc-edx) [inline oeps/oep-0037/decisions/0001-yaml-centralization.rst:7]: Any reason this one is Draft and the other two are Accepted?

Comment (jinder1s) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:7]: This should be Accepted.

Comment (jinder1s) [reply oeps/oep-0037/decisions/0001-yaml-centralization.rst:7]: Will change in future commit

File: oeps/oep-0037/decisions/0001-yaml-centralization.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Devstack currently has no standard way of creating dev data. Overtime, developers have figured out their own ways to create data for local development. Recently, `OEP-37`_ was proposed, which suggests a standardized method for creating dev data. We decided the proposal was good enough to warrant a prototype and during the prototyping phase, we ran into the question of how much of this method we want to centralize into devstack, vs keep things decentralized by keeping as much as is possible in each IDAs.
```
Comment (dianakhuang) [inline oeps/oep-0037/decisions/0001-yaml-centralization.rst:12]: typo: Overtime -> Over time

**Review by dianakhuang — APPROVED** [review]
typo fix, but that's it

**Pull request merged**